src/Mvc/test/WebSites/BasicWebSite/Controllers/ContactApiController.cs

```diff
@@ -131,11 +131,6 @@ private class TestModelBinder : IModelBinder
             public Task BindModelAsync(ModelBindingContext bindingContext)
             {
                 var val = bindingContext.ValueProvider.GetValue(bindingContext.ModelName);
-                if (val == null)
-                {
-                    return Task.CompletedTask;
-                }
-
```

Comment:
This is a semantic change.

Reply:
ValueProviderResult is a struct for result would always be false

Reply:
This is probably a bug then? /cc @dougbu

Reply:
@rynowak what exactly do you want my thoughts on?

Reply:
oh nvmt his is in test code. Just ignore then

Reply:
Is it OK to remove the if condition entirely (which doesn't change the current behavior) or should be replaced with something like `if (val.Length > 0)`?